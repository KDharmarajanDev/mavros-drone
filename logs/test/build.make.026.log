[35m[1mScanning dependencies of target offb_node[0m
[ 33%] [32mBuilding CXX object CMakeFiles/offb_node.dir/src/offb_node.cpp.o[0m
[01m[K/home/karthikdharmarajan/Documents/Power_Line_Inspection/src/test/src/offb_node.cpp:[m[K In function â€˜[01m[Kint main(int, char**)[m[Kâ€™:
[01m[K/home/karthikdharmarajan/Documents/Power_Line_Inspection/src/test/src/offb_node.cpp:108:45:[m[K [01;31m[Kerror: [m[Kâ€˜[01m[Kcurrent_pose[m[Kâ€™ was not declared in this scope
   float deltaX = abs(pose.pose.position.x - [01;31m[Kcurrent_pose[m[K.pose.position.x);
                                             [01;31m[K^~~~~~~~~~~~[m[K
[01m[K/home/karthikdharmarajan/Documents/Power_Line_Inspection/src/test/src/offb_node.cpp:108:45:[m[K [01;36m[Knote: [m[Ksuggested alternative: â€˜[01m[Kcurrent_state[m[Kâ€™
   float deltaX = abs(pose.pose.position.x - [01;36m[Kcurrent_pose[m[K.pose.position.x);
                                             [01;36m[K^~~~~~~~~~~~[m[K
                                             [32m[Kcurrent_state[m[K
CMakeFiles/offb_node.dir/build.make:62: recipe for target 'CMakeFiles/offb_node.dir/src/offb_node.cpp.o' failed
make[2]: *** [CMakeFiles/offb_node.dir/src/offb_node.cpp.o] Error 1
CMakeFiles/Makefile2:2057: recipe for target 'CMakeFiles/offb_node.dir/all' failed
Makefile:140: recipe for target 'all' failed
make[1]: *** [CMakeFiles/offb_node.dir/all] Error 2
make: *** [all] Error 2
