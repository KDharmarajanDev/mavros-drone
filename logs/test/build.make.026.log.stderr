[01m[K/home/karthikdharmarajan/Documents/Power_Line_Inspection/src/test/src/offb_node.cpp:[m[K In function â€˜[01m[Kint main(int, char**)[m[Kâ€™:
[01m[K/home/karthikdharmarajan/Documents/Power_Line_Inspection/src/test/src/offb_node.cpp:108:45:[m[K [01;31m[Kerror: [m[Kâ€˜[01m[Kcurrent_pose[m[Kâ€™ was not declared in this scope
   float deltaX = abs(pose.pose.position.x - [01;31m[Kcurrent_pose[m[K.pose.position.x);
                                             [01;31m[K^~~~~~~~~~~~[m[K
[01m[K/home/karthikdharmarajan/Documents/Power_Line_Inspection/src/test/src/offb_node.cpp:108:45:[m[K [01;36m[Knote: [m[Ksuggested alternative: â€˜[01m[Kcurrent_state[m[Kâ€™
   float deltaX = abs(pose.pose.position.x - [01;36m[Kcurrent_pose[m[K.pose.position.x);
                                             [01;36m[K^~~~~~~~~~~~[m[K
                                             [32m[Kcurrent_state[m[K
make[2]: *** [CMakeFiles/offb_node.dir/src/offb_node.cpp.o] Error 1
make[1]: *** [CMakeFiles/offb_node.dir/all] Error 2
make: *** [all] Error 2
